ouroboros-network/src/Ouroboros/Network/NodeToNode.hs

```diff
@@ -287,24 +292,29 @@ ipSubscriptionWorker
           peerid
           NodeToNodeProtocols
           IO BL.ByteString x y)
-    -> IO void
+    -> IO ()
```

Comment:
is removing of `void` here is necessary? or just a consequence of using `void` below